index.js

```diff
@@ -56,5 +56,37 @@ module.exports = {
         substrings: fieldsReplacements.map(i => i.substring)
       }
     };
+  },
+  /**
+   * Generates a script object which increments fields on the _source document
+   * @param {Object[]} fieldsReplacements Array of objects describing what to replace
```

Comment:
> {Object[]} fieldsReplacements Array of objects describing what to replace

Shouldn't this be a `fieldsMap` object instead of array of objects?

Reply:
Absolutely!